.github/workflows/deploy.yaml

```diff
@@ -24,7 +24,7 @@ jobs:
         run: npm run test:ci
 
       - name: Create build
-        run: npm run build
+        run: ng build --configuration production --base-href=/OsmGo/
```

Comment:
We do not have installed the Angular CLI in this context. Either we should install it first or we can use `npm run ng -- --configuration production --base-href=/OsmGo/` instead.

Reply:
Thank you for the code review.
Yes, sorry, I had forgotten about the paths of webmanifest and some custom SVG.

I don't understand the double '--'. Do you have any documentation about this?

Reply:
The double `--` is for arguments that are not meant to be consumed by `npm` directly but are passed to the script that is called. 

From `npm help run-script`:

```
NAME
       npm-run-script - Run arbitrary package scripts

   Synopsis
         npm run-script <command> [--if-present] [--silent] [-- <args>]
         [...]

   Description
       [...]

       Any positional arguments are passed to the specified script.  Use -- to pass --prefixed flags and options which would otherwise be parsed by npm.

       For example:

         npm run test -- --grep="pattern"

       The arguments will only be passed to the script specified after npm run and not to any pre or post script.
```

See also [this page](https://dev-tips.com/nodejs/pass-custom-command-line-arguments-to-npm-scripts) for explanation.